.github/workflows/development.yml

```diff
@@ -114,6 +114,7 @@ jobs:
               "PORT=${{ secrets.PORT }}" \
               "NODE_ENV=${{ secrets.NODE_ENV }}" \
               "TZ=${{ secrets.TZ }}" \
+              "LOG_LEVEL=${{ secrets.LOG_LEVEL }}" \
```

Comment:
Is this the right change to make for the infrastructure @abhi-markan?

Reply:
@lmarrai-sw  do you have access to `Vars` in the repo settings?

Reply:
@IgnatG unfortunately not, I don't have access to `Settings` at all for the repo. Abhi's added the variable for me though

@abhi-markan I've changed `secrets` to `vars`